src/features/global/CollapsePossiblyRelatedThreads.js

```diff
@@ -8,7 +8,7 @@ class CollapsePossiblyRelatedThreads extends Feature {
       section: Global,
       name: "Collapse Possibly Related Threads",
       default: false,
-      description: "Automatically collapse 'Possibly Related Threads' results.\n\nClick the toggle image to view results"
+      description: "Automatically collapse 'Possibly Related Threads' results.\n\nClick the toggle image to view results."
```

Comment:
I deleted this file. Did you pull my branch before you made these changes?

Reply:
You forgot to commit the deletion then.